### DISCUSSION TITLE
for vector source, dose it support using AWS EC2 system log ?

### DISCUSSION THREAD

**Discussion by foreversunyao** [vectordotdev/vector#15185] [Q&A] [answered]
for AWS EC2 , https://docs.aws.amazon.com/managedservices/latest/userguide/access-to-logs-ec2.html
```
/var/log/audit/audit.log
/var/log/cron
/var/log/amazon/ssm/amazon-ssm-agent.log
/var/log/secure
/var/log/aws/ams
/var/log/maillog
/var/log/yum.log
/var/log/messages
/var/log/cloud-init-output.log
/var/log/cloud-init.log (Amazon Linux 1 / Amazon Linux 2 only)
```

I dont see any existing sources proper ? https://vector.dev/docs/reference/configuration/sources/

Thanks

**Comment by jszwedko** [comment, marked as answer]
Hey @foreversunyao ! I'm not sure I completely understand the question, but if you just want to read those files with Vector, the `file` source is what you would be after.

**Reply by foreversunyao** [reply]
Thank you for answering the question, I missed it( I thought there was some specific thing for ec2 logs source). I can solve my problem with the **file** source.  : )
